### DISCUSSION TITLE
grapesjs-plugin-export file name does not change from grapesjs_template_ to custom name

### DISCUSSION THREAD

**Discussion by codingmachine16** [GrapesJS/grapesjs#4085] [Q&A]
Hi @artf 

    When I export template as zip, it saves grapesjs_template_ . How can I use a custom name. 

I am using grapesjs-preset-webpage plugin and when I try to change filenamePfx using below code . It does not work for me.

      plugins: ['grapesjs-plugin-export'],
      pluginsOpts: {
        'grapesjs-plugin-export': { filenamePfx: 'custome_Name_' }
      }

Please help me . Thanks

**Comment by sajjadhussain14** [comment]
Please help any
